src/main/java/net/sf/jabref/gui/keyboard/KeyBindingCategory.java

```diff
@@ -5,19 +5,19 @@
 public enum KeyBindingCategory {
 
     FILE(
-            Localization.menuTitle("File")),
+            Localization.lang("File")),
```

Comment:
Why this change? Isn't the category mirroring the menu structure? Menu strings are translated in the menu translation files, not the normal language files.